### PR TITLE
docs(guides): begin adding documentation for gdml to gltf convertion

### PR DESCRIPTION
**Pull request by andrewpap22**
Hello all, 

I have started adding the documentation about the gdml to root and gltf convention to be displayed properly into phoenix at this place. I have to add a couple more things in the next commits, of course you can see that it is not completed but I wanted to ask if the place that I've added it looks fine for you? 

Cheers, 

Andreas

### CODE REVIEW

**Comment by EdwardMoyse** [comment]
This looks great! Looking forward to seeing the next commits (I might have  someone I would like to share this with)

**Comment by andrewpap22** [comment]
Hello all, 

After the checks get done successfully (hopefully) this should be good to go I think. 

@EdwardMoyse and @9inpachi you might want to have a look as well before merging! 

Cheers!

File: guides/developers/convert-gdml-to-gltf.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+3. Do wait a couple of seconds and it should load automatically and you'll be able to see it visualized properly. 
+4. `If you don't know where to find the root object name just scroll a bit down on the page and you'll find relevant info.` 
+5. Go at the bottom of the page and press the button `export to gltf` (only if you import your root file via a URL. Importing it from your local machine will translate it automatically to gltf)
+6. It might take a couple of seconds or minutes (depending on the file size) and then your .gltf file will get downloaded locally automatically. 
+7. Upload your .gltf file to the GLTF viewer at the bottom of the page and verify that it was extracted properly. 
+8. Go and visualize your file inside Phoenix and play with it! 
+
+> Here's a gif demonstrating the above steps
+
+![gltf-exporter](./../images/output.gif)
```
Comment (9inpachi) [inline guides/developers/convert-gdml-to-gltf.md:62]: The gif seems to have a low resolution. Hard to follow what's happening inside it.

File: guides/developers/convert-gdml-to-gltf.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+### Split your GLTF geometry into multiple parts
+
+After extracting our geometry and visualizing into Phoenix we might want to split it into multiple subparts to visualize and play with each part of our geometry seperately. 
+
+To do that one might need to modify some code for his .gltf file but extensive documentation is written for the ease of that purpose over 👉 [here](https://github.com/andrewpap22/root_cern-To_gltf-Exporter)!
+
+Just follow the documentation in the above link and you should be good to split your whole geometry into multiple subparts and be able to visualize into Phoenix each one of them seperately or all together with the option to enable and disable each part from the UI!
+
+Again feel free to open an [issue](https://github.com/HSF/phoenix/issues) if stuck and someone friendly folk will try and help!
```
Comment (9inpachi) [inline guides/developers/convert-gdml-to-gltf.md:72]: NIT: and some friendly folk will try and help!*

**Review by 9inpachi — COMMENTED** [review]
Just two comments. Everything else looks good.

File: guides/developers/convert-gdml-to-gltf.md (hunk 1 above)
Comment (andrewpap22) [reply guides/developers/convert-gdml-to-gltf.md:62]: Ah alright, I'll try to make it larger with greater resolution, thanks!

Comment (andrewpap22) [reply guides/developers/convert-gdml-to-gltf.md]: Done! It should be slightly better in the default preview of the markdown in the GitHub page but still not 100% readable. But if you open the gifs in a new tab in the browser, they should be at full resolution and 100% visible and readable!

**Comment by andrewpap22** [comment]
I think this is ready to go! I think you can even merge it now, as it is only markdown text in a standalone file and it doesn't matter if the tests from master fail, or ping me when you have the tests fixed and ready, so I can fetch from master and then it should be completely ready to go.

**Review by 9inpachi — APPROVED** [review]
